F21/cemilcan98/app.py

```diff
@@ -0,0 +1,84 @@
+def toLowerCase(i):

```

Comment:
I would suggest a more descriptive parameter name here.

Reply:
I have just pushed a new version of the code with considering all your comments and suggestions.